### ISSUE TITLE
Vulpia microstachys

### ISSUE THREAD

**Issue by brouille** [Canadensys/vascan-data#375]
[Originally posted on GoogleCode (id 1601) on 2012-12-14]

http://data.canadensys.net/vascan/taxon/8005

Marc et Marilyn:

J'ai dû mettre en synonymie les deux variétés de ce taxon sous l'espèce. Svp:

fr: pas de vernaculaire français pour l'espèce - j'ai éliminé ceux des var. car non appropriés

en: pls check that the name currently accepted is correct and that the vernaculars coming from the vars. are worth keeping

Merci
Luc

**Comment by MFavreau** [comment]
[Originally posted on GoogleCode on 2012-12-14 17:43Z]

Accepted French common name added.

I also entered, as a synonym, the name « petite vulpie » that I had proposed several years ago for the species as a whole and which was used a few times in various reports. Given the fact that the habit and the inflorescence of this species are not particularly smaller than those of other Vulpia, I preferred to use « vulpie du Pacifique » as an accepted French name. It is the only native species of Vulpia with a range basically restricted to the Pacific watershed.
